lib/my_api_client/request/basic.rb

```diff
@@ -20,10 +20,18 @@ module Basic
           # @param body [Hash, nil]
           #   Request body. You should not specify it when use GET method.
           # @return [Sawyer::Resource]
-          #   Response body instance.
+          #   Response body instance if the block is not given.
+          # @yield
+          #   Process the response body with the given block.
+          # @yieldparam response [Sawyer::Response]
+          #   Response instance.
+          # @yieldreturn [<T>]
+          #   The block result.
+          # @return [<T>]
+          #   Whatever the block returns if the block is given.
```

Comment:
`<T>` is not ruby syntax. I think should be used `Object`.

```suggestion
          # @yieldreturn [Object]
          #   The block result.
          # @return [Object]
          #   Whatever the block returns if the block is given.
```

Reply:
Fixed on 814764837e7d1e7c747c7bf056839d3c1c132fb3.